SUMMARY.md

```diff
@@ -2,6 +2,11 @@
 
 * [Introduction](README.md)
 
+## Community
+
+* [Roles within Mozilla l10n communities](community/l10n_community_roles.md)
```

Comment:
We should try to keep these short, since they show up in the sidebar. What about using just "Roles" and "Reviewing translations"?

Reply:
Works even better (and I don't think it's mandatory to keep title and sidebar link in sync)